README.md

```diff
@@ -1 +1,94 @@
-# Hello-world-
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Author](#author)
+
+# 📖 Hello-world microverse project
+
+this is a Hello-world microverse project that shows its heading
+
+## 🛠 Built With <a name="built-with"></a>
+
+-Html
+-css
+
+### Tech Stack <a name="tech-stack"></a>
+
+-html,css, linters git and github
+
+### Key Features <a name="key-features"></a>
+
+- Hello-world Microverse heading
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- To have node js installed on your machine.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+cd my-folder
+
+- clone the repository:
+  git clone https://github.com/ismayito/Hello-world-.git
+
+### Install
+
+Install this project with:
+
+- change to the clone folder for example:
+- run npm install to install dependencies of this project.
+
+### Usage
+
+To run the project, execute the following command:
+
+- open the index.html file in the browser then you will have the project up and running
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://https://github.com/ismayito)
+- Twitter: [@twitterhandle](https://twitter.com/@IsmailMayito)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mayito-ismail-2b0067178/)
+
+🔭 Future Features
+
+- Hero section
+- Footer section
+- Call to action section
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+📝 License
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```

Comment:
Good job 😄 . But as the previous reviewer said, your readme file doesn't have an MIT license, kindly add an MIT license to your project as this can give your project a more professional look😉. You may delete the `red` section, and create the `MIT.md` file, you may use this as an [example](https://github.com/joel-kalema/Hello-Microverse/blob/main/MIT.md). Then assign that file to the `MIT` selected in bleu. 👍 
![Annotation 2023-02-14 172047](https://user-images.githubusercontent.com/57408419/218806899-fb7f1464-3d57-4df3-8b88-1b45a2ffda50.jpg)

Reply:
> Good job 😄 . But as the previous reviewer said, your readme file doesn't have an MIT license, kindly add an MIT license to your project as this can give your project a more professional look😉. You may delete the `red` section, and create the `MIT.md` file, you may use this as an [example](https://github.com/joel-kalema/Hello-Microverse/blob/main/MIT.md). Then assign that file to the `MIT` selected in bleu. 👍 ![Annotation 2023-02-14 172047](https://user-images.githubusercontent.com/57408419/218806899-fb7f1464-3d57-4df3-8b88-1b45a2ffda50.jpg)

how do I assign the  MIT.md file to wait

Reply:
Hello @ismayito. 😃, You may add the path, it'll be something this `[MIT](./MIT.md)` 👍 
 - or  `This project is [MIT](./MIT.md) licensed.`